config.yml

```diff
@@ -9,6 +9,9 @@ channel_mappings:
   "#bottest chanKey": 316038111811600387
   "#bottest2": 318327329044561920
 
+# irc_ignores_discord:
+#  MEE6: true
```

Comment:
i think this should be a `[]string` instead of a string map.

Reply:
I would have thought that as well, but the issue is, can we do a fast lookup in a list? Whereas a map is going to yield generally consistent lookup times.

Reply:
we can still keep it a map for easy lookup in the `Config` struct, but I think when pulling it out of viper, we should expect the input to to be a list.

Reply:
Ah, read in from config as list, and parse out into a map later in the configuration func. That's actually a good way to go about it. I'll look into doing that soonish.

Reply:
thank you!